README.md

```diff
@@ -149,6 +149,7 @@ matters to you.
 * <a href="https://github.com/fastify/fastify/blob/master/docs/Reply.md"><code><b>Reply</b></code></a>
 * <a href="https://github.com/fastify/fastify/blob/master/docs/Request.md"><code><b>Request</b></code></a>
 * <a href="https://github.com/fastify/fastify/blob/master/docs/Error-Handling.md"><code><b>Error Handling</b></code></a>
+* <a href="https://github.com/fastify/fastify/blob/master/docs/Errors.md"><code><b>Errors</b></code></a>
```

Comment:
Maybe we can rename the title here to `Error Codes` to be more specific

Reply:
Also change the filename to `Error-Codes.md`?

Reply:
I am not sure about changing the name of the file. @mcollina wdyt?

Reply:
I’m ok with the change